picard/ui/coverartbox.py

```diff
@@ -33,7 +33,7 @@ class ActiveLabel(QtGui.QLabel):
     """Clickable QLabel."""
 
     clicked = QtCore.pyqtSignal()
-    imageDropped = QtCore.pyqtSignal(QtCore.QUrl)
+    imageDropped = QtCore.pyqtSignal(QtCore.QUrl, QtCore.QByteArray)
```

Comment:
Please use snake case for all variable names. See https://github.com/metabrainz/picard/blob/master/CONTRIBUTING.md